bbc/simorgh
# [WSTEAM1-300] Expand download functionality to all podcast services

Resolves JIRA [300]

Overall changes
======
Expands the download functionality to all podcast services (previously only on Burmese service)

Code changes
======
- Update tests and snapshots
- Expand download functionality to all Podcast services
- Add download translations to `PodcastExternalLinks`
- Add download translations to config

Testing
======
1. Pull down this PR
2. Go to a podcast page: http://localhost:7080/arabic/podcasts/p02pc9qc or http://localhost:7080/hausa/podcasts/p08mlgcb
3. Observe that these services now have a 'download' button within the podcast external links section

Helpful Links
======
[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
